sympy/printing/pretty/pretty_symbology.py

```diff
@@ -615,6 +615,20 @@ def annotated(letter):
     else:
         return ascii_pics[letter]
 
+def is_combining(sym):
+    """Check whether symbol is a unicode modifier.
+
+    See stringPict.width on usage.
+    """
+    if not pretty_use_unicode():
+        return False
```

Comment:
Why is this here? This is the reason that the test failures are not reproducible across environments. I don't see why this check should change the return value of this function

Reply:
Agreed. This function shouldn't be concerned be with what `pretty_use_unicode`'s output is. I have removed it and it is working correctly now on `pytest`.